x-pack/legacy/plugins/siem/public/components/page/hosts/hosts_table/index.test.tsx

```diff
@@ -43,10 +43,14 @@ describe('Load More Table Component', () => {
               data={mockData.Hosts.edges}
               id="hostsQuery"
               indexPattern={mockIndexPattern}
-              hasNextPage={getOr(false, 'hasNextPage', mockData.Hosts.pageInfo)!}
+              fakeTotalCount={getOr(50, 'fakeTotalCount', mockData.Hosts.pageInfo)!}
```

Comment:
I have great news here ... We have reduced our usage of `!` (non-null assertion operator) in so many areas naturally as the code base has matured and because of these great changes you are free to remove the "bang/!" and get rid of it here and the other spots in this file when you want to:

<img width="706" alt="Screen Shot 2019-07-19 at 12 39 22 PM" src="https://user-images.githubusercontent.com/1151048/61557840-847e7200-aa22-11e9-9ea4-47f74516abd2.png">
 
I tested it out locally and it looks work well! It will give you more type guarantees without it.